README.md

```
@@ -121,6 +137,12 @@ To install with [Carthage](https://github.com/Carthage/Carthage), simply add thi
 github "efremidze/Haptica"
 ```
 
+// For projects that support iOS 9
+.package(url: "https://example.com/your-package", from: "1.0.0")
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)